src/element/newElement.ts

```diff
@@ -31,29 +51,36 @@ export function newElement(
     strokeWidth,
     roughness,
     opacity,
-    seed: randomSeed(),
-    points: [] as readonly Point[],
-    version: 1,
-    versionNonce: 0,
-    isDeleted: false,
+    seed: rest.seed ?? randomSeed(),
+    version: rest.version || 1,
```

Comment:
Why not `??`?

Reply:
Versions are 1-based AFAIK, but I guess we can be a bit defensive here :+1:

Reply:
Or should we... I think we used to have 0-based versions, and this way it'll migrate it to 1-based.

Reply:
So I'll keep it.

Reply:
yes, versions need to be 1-based now due to how `getDrawingVersion()` works